index.html

```diff
@@ -100,27 +100,27 @@ <h2 class="V-main-h2">Get started any way you&nbsp;want</h2>
       <div class="section1-part1">
         <div class="icon1"><img src="assets/images/icon-section1-part1.png" alt="icon-section1-part1"></div>
         <h3 class="semibold1"> Install via package manager </h3>
-        <p class="p-s1-p1" >
+        <p class="p-s1-p1 paragraph" >
```

Comment:
class name `paragraph` is a bad one . please choose another word for naming. In reason `p` means `paragraph` paragraph with class `paragraph`.... Too much paragraphs :D

Reply:
done